src/pages/billing/SpendReport/SpendReport.test.ts

```diff
@@ -182,8 +182,33 @@ describe('SpendReport', () => {
     })
     expect(screen.getByTestId('spend')).toHaveTextContent('$1,110.17*')
     expect(getSpendReport).toHaveBeenCalledTimes(2)
-    expect(getSpendReport).toHaveBeenNthCalledWith(1, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-03-02' })
-    expect(getSpendReport).toHaveBeenNthCalledWith(2, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-01-01' })
+    expect(getSpendReport).toHaveBeenNthCalledWith(1, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-03-02', aggregationKeys: ['Workspace~Category', 'Category'] })
+    expect(getSpendReport).toHaveBeenNthCalledWith(2, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-01-01', aggregationKeys: ['Workspace~Category', 'Category'] })
+  })
+
+
+  it('displays cost information', async () => {
```

Comment:
I think this is a bad merge-- I moved this test case up.

Reply:
Yeah, looks like I duplicated it accidentally when rebasing